chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 0.9.1
+version: 0.9.2
 appVersion: DEVEL
```

Comment:
not part of this change, but should this be `v1.0.0-beta.3`? /cc @prydonius @migmartri